openpype/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -28,12 +28,36 @@ class CreateTextures(Creator):
     icon = "picture-o"
 
     default_variant = "Main"
+    image_format = None
+    exportPresetUrl = None
+    exportSize = None
+    exportPadding = "infinite"
+    exportDilationDistance = 16
 
     def create(self, subset_name, instance_data, pre_create_data):
 
         if not substance_painter.project.is_open():
             raise CreatorError("Can't create a Texture Set instance without "
                                "an open project.")
+        self.exportPresetUrl = pre_create_data.get("exportPresetUrl",
+                                                   self.exportPresetUrl)
+        instance_data["exportPresetUrl"] = self.exportPresetUrl
+
+        self.image_format = pre_create_data.get("exportFileFormat",
+                                                self.image_format)
+        instance_data["exportFileFormat"] = self.image_format
+
+        self.exportSize = pre_create_data.get("exportSize",
+                                              self.exportSize)
+        instance_data["exportSize"] = self.exportSize
+
+        self.exportPadding = pre_create_data.get("exportPadding",
+                                                 self.exportPadding)
+        instance_data["exportPadding"] = self.exportPadding
+
+        self.exportDilationDistance = pre_create_data.get("exportDilationDistance",
+                                                          self.exportDilationDistance)
+        instance_data["exportDilationDistance"] = self.exportDilationDistance
```

Comment:
```suggestion
    # Transfer settings from pre create to instance
    for key in [
        "exportPresetUrl",
        "exportFileFormat",
        "exportSize",
        "exportPadding",
        "exportDilationDistance"
    ]:
        if key in pre_create_data:
            instance_data[key] = pre_create_data[key]
```
Maybe better like this?

Reply:
Yes!